spinalcordtoolbox/download.py

```diff
@@ -375,13 +376,25 @@ def install_named_dataset(dataset_name, dest_folder=None, keep=False):
 
 def list_datasets():
     """
-    :returns: A table containing listing the downloadable datasets
+    :returns: A table listing the downloadable datasets
     :rtype: str
     """
+    color = {True: 'LightGreen', False: 'LightRed'}
     table = f"{'DATASET NAME':<30s}{'TYPE':<20s}\n"
     table += f"{'-' * 50}\n"
-    sortedDatasets = sorted(DATASET_DICT, key=lambda k: DATASET_DICT[k]['download_type'])
+    sortedDatasets = sorted(DATASET_DICT,
+                            key=lambda k: DATASET_DICT[k]['download_type'])
     for dataset_name in sortedDatasets:
         download_type = DATASET_DICT[dataset_name]['download_type']
-        table += f"{dataset_name:<30s}{download_type:<20s}\n"
+        dataset_status = dataset_name.ljust(30)
+        if download_type != "Binaries":
+            path_dataset = DATASET_DICT[dataset_name]['default_location']
+            installed = os.path.exists(path_dataset)
```

Comment:
Another nitpick: Perhaps we could also check if the folder is non-empty, by doing:

```suggestion
            installed = (os.path.exists(path_dataset)
                         and len(os.listdir(path_dataset)) > 0):
```

Still not a foolproof check, but it covers the case where the dataset folder was created but the actual downloading/unzipping process failed somehow.